protos/runner_services_pb.rb

```diff
@@ -0,0 +1,21 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# Source: runner.proto for package 'fastlane_c_i'
+
+require "grpc"
+require "runner_pb"
+
+module FastlaneCI
+  module Runner
+    class Service
+      include GRPC::GenericService
+
+      self.marshal_class_method = :encode
+      self.unmarshal_class_method = :decode
+      self.service_name = "fastlane_c_i.Runner"
+
+      rpc :Spawn, Command, stream(Log)
```

Comment:
What does this do, is that the `Command` we defined in the proto pufs?

Reply:
Yes, this is the ruby struct defined in the proto. I don't quite understand this syntax, but I read it as:
define an rpc method on this service called `Spawn` that accepts a `Command` and returns a stream of `Log`s